package.json

```diff
@@ -98,9 +100,12 @@
     "fancy-log": "^1.3.3",
     "firebase": "^7.14.6",
     "firebase-tools": "^9.3.0",
-    "gorko": "^0.7.0",
+    "gorko": "^0.7.1",
     "gts": "^2.0.2",
     "gulp": "^4.0.0",
+    "gulp-clean-css": "^4.3.0",
+    "gulp-header": "^2.0.9",
```

Comment:
I think clean-css can be removed. Is gulp-header used anywhere or should we remove it as well?

Reply:
Ah good catch!